docs/core/MONOREPO.md

```diff
@@ -30,15 +30,18 @@ The purpose of this monorepo is to host various **[npm packages ↗](https://doc
       "install:nuke": "rm -rf pnpm-lock.yaml && pnpm install:fresh"
```

Comment:
We need to clear out more of the `pnpm` stuff like this line `rm -rf pnpm-lock.yaml && pnpm install:fresh`. I can help with that

Reply:
Whoops! Will get to it now